GameOfLife/validator.h

```diff
@@ -0,0 +1,22 @@
+#ifndef VALIDATOR_H
+#define VALIDATOR_H
+#include <string>
+#include "size.h"
+
+int const frame = 2;
+
+class Validator
+{
+public:
+    Validator();
+    bool validate(std::string);
+    void CouterOfFirstLineWidth(std::__cxx11::string readString, int &lineWidth);
+    Size getSizeOfBoard() const;
```

Comment:
Validator is counting lenght of row to compare it with other rows. As it counts, it also set Size in case of positive result of validation.